generator/src/test/java/com/scottlogic/deg/generator/cucumber/EqualTo.feature

```diff
@@ -283,33 +286,30 @@ Scenario: Running an 'equalTo' request that includes the number zero should be s
 Scenario: Running an 'equalTo' request that includes a date value (not a string) should be successful
      Given there is a field foo
        And foo is equal to 2010-01-01T00:00:00.000
-     Then the following data should be generated:
+     Then the following data is generated:
        | foo                     |
        | 2010-01-01T00:00:00.000 |
 
 Scenario: Running an 'equalTo' request that includes a date value (leap year) should be successful
      Given there is a field foo
        And foo is equal to 2020-02-29T00:00:00.000
-       And foo is of type "temporal"
      Then the following data should be generated:
        | foo                     |
        | 2020-02-29T00:00:00.000 |
 
 Scenario: Running an 'equalTo' request that includes a date value (earliest date) should be successful
      Given there is a field foo
-       And foo is equal to 0001-01-01T00:00:01.000
-       And foo is of type "temporal"
+       And foo is equal to 0001-01-01T:00:00:01.000
      Then the following data should be generated:
        | foo                      |
-       | 0001-01-01T00:00:01.000 |
+       | 0001-01-01T:00:00:01.000 |
```

Comment:
There are lots of places where there is a `:` immediately after the `T` which is not valid for the expression of a date/time